_FIPS201/federation.md

```diff
@@ -15,7 +15,7 @@ Federation protocols allow a trusted IdP to assert a cardholder's identity to an
 
 ## 7.1 Connecting PIV to Federation {#s-7-1}
 
-When using a federation protocol, the PIV Card or derived PIV credential is not directly presented to the relying subsystem. Instead, the PIV Card or derived PIV credential SHALL be used to authenticate the PIV cardholder to the IdP of a federation system.[^issuer] The IdP SHALL associate this login with the PIV account of the cardholder and SHALL create an assertion representing the cardholder to be sent to the RP, including attributes of the cardholder stored in the PIV account. Upon receipt, the RP SHALL validate the assertion and use the attributes provided in the assertion to match the cardholder information to the information on record, as discussed in [Section 3.1.3](system.md#s-3-1-3). The connections and components of a federated protocol are shown in [Figure 3-4](system.md#fig-3-4).
+When using a federation protocol, the PIV Card or derived PIV credential is not directly presented to the relying subsystem. Instead, the PIV Card or derived PIV credential SHALL be used to authenticate the PIV cardholder to the IdP of a federation system.[^issuer] The IdP SHALL associate this authenticated session with the PIV account of the cardholder and SHALL create an assertion representing the cardholder to be sent to the RP, including attributes of the cardholder stored in the PIV account. Upon receipt, the RP SHALL validate the assertion and use the attributes provided in the assertion to match the cardholder information to the information on record, as discussed in [Section 3.1.3](system.md#s-3-1-3). The connections and components of a federated protocol are shown in [Figure 3-4](system.md#fig-3-4).
```

Comment:
```suggestion
When using a federation protocol, the PIV Card or derived PIV credential is not directly presented to the relying subsystem. Instead, the PIV Card or derived PIV credential SHALL be used to authenticate the PIV cardholder to the IdP of a federation system.[^issuer] The IdP SHALL associate this authentication event with the PIV account of the cardholder and SHALL create an assertion representing the cardholder to be sent to the RP, including attributes of the cardholder stored in the PIV account. Upon receipt, the RP SHALL validate the assertion and use the attributes provided in the assertion to match the cardholder information to the information on record, as discussed in [Section 3.1.3](system.md#s-3-1-3). The connections and components of a federated protocol are shown in [Figure 3-4](system.md#fig-3-4).
```
I think it's a little confusing to refer to the creation of an authenticated session between the cardholder and IdP, even though technically that may be what is happening under the covers. The session that matters is the cardholder-RP session, and it's really the event of authentication that causes the creation of an assertion, so I'd prefer if we emphasize that.

Reply:
My original goal was to draw out the fact that there's a separate session at the IdP, but given this context it's the event that matters so I'm fine with this edit.